### DISCUSSION TITLE
Disable Backup Notification

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#985] [Ideas] [answered]
It would be nice if there would be an option to disable this notification:
![grafik](https://user-images.githubusercontent.com/75573284/181908316-3b56d621-f838-45cd-9252-bcb12f28d44b.png)
So that I only get a notification if it fails, that would be nice.

**Comment by szaimen** [comment]
See https://github.com/nextcloud/all-in-one/discussions/484#discussioncomment-2566923

**Comment by szaimen** [comment, marked as answer]
Will be possible with https://github.com/nextcloud/all-in-one/pull/3516

**Reply by szaimen** [reply]
This is now released with v7.5.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
